internal/app/fhir/r5/helpers.go

```diff
@@ -65,6 +66,7 @@ func newXhtml(value string) *datatypespb.Xhtml {
 	return &datatypespb.Xhtml{Value: value}
 }
 
+//nolint:gosec
```

Comment:
it's complaining that the cast might overflow - do we know if this is a potential real issue in practice?

Reply:
fine in practice